src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralService.kt

```diff
@@ -36,10 +36,12 @@ import java.time.OffsetDateTime
 import java.util.UUID
 import javax.transaction.Transactional
 
-data class ContactableProbationPractitioner(
-  val firstName: String,
-  val email: String,
-)
+data class ResponsibleProbationPractitioner(
+  override val firstName: String,
+  override val email: String,
+  val staffId: Long?,
```

Comment:
perhaps it would help a bit to rename this to `deliusStaffId` to aid in understanding where this id is from

Reply:
yeh that's a good idea - thanks